stacks/dotnet50/devfile.yaml

```diff
@@ -2,12 +2,12 @@ schemaVersion: 2.1.0
 metadata:
   name: dotnet50
   displayName: .NET 5.0
-  description: Stack with .NET 5.0
+  description: MVC .NET 5.0 starter application
```

Comment:
The stacks are different from the sample entires in the extraDevfileEntries.yaml. The description should be focusing on the type of application that we are supporting, not on the starter project application itself.